timbuctoo-instancev4/src/test/java/nl/knaw/huygens/timbuctoo/v5/dataset/DataSetFactoryTest.java

```diff
@@ -95,7 +99,7 @@ public void dataSetExistsReturnsFalseIfTheUserIsNotKnown() {
 
   @Test
   public void dataSetExistsReturnsFalseIfTheUserDoesNotOwnADataSetWithTheDataSetId() throws DataStoreCreationException {
-    dataSetFactory.createImportManager("otherOwner", "dataSetId");
+    getImportManager("ownerId", "otherDataSetId");
```

Comment:
It looks like this test could be interpreted two ways.
Which one is more explanatory?